readme.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
 1. Replace `<your_account>` with your Github username in the links
-    - [DEMO LINK](https://<your_account>.github.io/layout_hello-world/) <br>
-    - [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+    - [DEMO LINK](https://<stas-dukach>.github.io/layout_hello-world/) <br>
+    - [TEST REPORT LINK](https://<stas-dukach>.github.io/layout_hello-world/report/html_report/)
```

Comment:
Read the comment:
![image](https://user-images.githubusercontent.com/60885495/111980835-852cad00-8b0f-11eb-9053-f930ef12c3ff.png)